emulsify.drush.inc

```diff
@@ -0,0 +1,299 @@
+<?php
+
+/**
+ * @file
+ * Contains Drush hooks. Inspired by Aeon drush commands.
+ */
+
+/**
+ * Implements hook_drush_command().
+ */
+function emulsify_drush_command() {
+  $items = array();
+
+  $items['emulsify'] = array(
+    'description' => 'Create an Emulsify theme.',
```

Comment:
I think the phrase "Emulsify-based" would be good. I'm open to discussion/opinions though.